core/rpc/mu-rpc.cabal

```diff
@@ -28,7 +28,8 @@ library
     Mu.Server
 
   build-depends:
-      base              >=4.12  && <5
+      aeson
```

Comment:
Please introduce bounds for this dependency. The best is to copy them from `mu-schema.cabal`.

Reply:
Done! Have you had issues with unbounded aeson versions in the past? I assumed stack would handle it.

Reply:
You are 100% right. However, we are trying not to leave Cabal-only users out, and for those we want to follow the PVP and write bounds as much as we can.